corehq/apps/reports/models.py

```diff
@@ -441,6 +466,11 @@ def get_report_content(self, attach_excel=False):
             })
             return _("An error occurred while generating this report."), None
 
+    @property
+    def is_configurable_report(self):
+        from corehq.apps.userreports.reports.view import ConfigurableReport
+        return type(self._dispatcher) == ConfigurableReport
```

Comment:
might be more extensible as `isinstance(self._dispatcher, ConfigurableReport)` (in case we ever subclass it)